client/src/app/api/models.ts

```diff
@@ -704,3 +704,13 @@ export type HubFile = {
   name: string;
   path: string;
 };
+
+export interface Questionnaire {
```

Comment:
Should we name this `CustomQuestionnaire` to distinguish it from `PathfinderQuestionnaire` rather than imply inheritance between the two? or is it intended that all questionnaires will eventually use this type and pathfinder objects will be deprecated?

Reply:
if pathfinder will be removed eventually and we'll only have one questionnaire type, i'm fine with leaving the name as-is.